src/pipeline-service/templates/_helpers.tpl

```diff
@@ -60,3 +60,7 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{- define "pipeline-service.pullSecrets" -}}
```

Comment:
can we define a common template in platform/templates/_helpers.tpl ?

{{- define "pipeline-service.pullSecrets" -}}
{{ include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.waitForInitContainer.image) "global" .Values.global ) }}
{{- end -}}

instead of defining in every service